lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -343,11 +343,13 @@ def _msd_diffusion_covariance(max_lags, n, intercept, slope):
     return base_covariance + localization_part
 
 
-def _diffusion_ols(mean_squared_displacements, num_points):
+def _diffusion_ols(lag_idx, mean_squared_displacements, num_points):
     """Estimate the intercept, slope and standard deviation of the slope based on the msd [2]_
 
     Parameters
     ----------
+    lag_idx : array_like
+        lag frame differences
```

Comment:
I find the description a bit confusing. Is this supposed to be the lag time in frames rather than seconds?

Reply:
Fair point. I renamed it to `lag time in frames`.